test_data/generated_test_data/ontologyR2RV2/knoraApiOntologySimple.jsonld

```diff
@@ -1267,6 +1283,14 @@
             },
             "@id": "knora-api:hasComment"
         },
+        {
+            "@id": "knora-api:hasCopyrightAttribution",
+            "@type": "owl:DatatypeProperty",
+            "knora-api:objectType": {
```

Comment:
Non-blocking: Is there a reason why you did not specify `knora-api:subjectType`?
I don't remember any more. Is there a plan to use this property also for our resources or will it only ever be for assets I would suggest: `knora-api:subjectType knora-api:FileValue`

Reply:
Yes, the subject type can either be a knora-admin:knoraProject or a knora-base:FileValue.

As this is not really evalutated and I do not know how to express this I left it open.